routes/messages/commands/amazon_link.js

```diff
@@ -0,0 +1,36 @@
+const Command = require('../classes/Command.js');
+const Books = require('../../../models/db/books.js');
+
+module.exports = async event => {
+  const id = 1; //need a dynamic way to pull ids
```

Comment:
Still a problem we need to solve.  We may have to hide the data in the string and cut it up to get what we need.